exercises/concept/coordinate-transformation/.docs/instructions.md

```diff
@@ -18,6 +21,9 @@ const result = moveCoordinatesRight2Px(4, 8);
 
 Implement the `scale2d` function that returns a function making use of a closure to perform a repeatable 2d scale of a coordinate pair.
 
+> In geometry, uniform [scaling][wiki-scale] refers to enlarging or shrinking vectors or shapes in the same direction. 
+> It can be interpreted as multiplying every point by a constant (scaling factor).
+> 
```

Comment:
Ahh, now it breaks the styling check. 
I think it doesn't like the empty line between the two quotes

Reply:
That's okay. Let's see if we can auto fix it.

Reply:
magic spells i see.
Good